guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -48,11 +48,11 @@ The base recommended external database configuration adjustments are as follows:
 listen_addresses = '*'
 ----
 +
-. Edit the `/var/lib/pgsql/data/pg_hba.conf` file:
+. Edit the `{postgresql-conf-dir}/pg_hba.conf` file:
 +
 [options="nowrap" subs="verbatim,quotes"]
 -----
-# vi /var/lib/pgsql/data/pg_hba.conf
+# vi {postgresql-conf-dir}/pg_hba.conf
```

Comment:
this doesn't get replaced, you need those magic options to the codeblock :roll_eyes: